contracts/Rad.sol

```diff
@@ -13,9 +13,4 @@ contract Rad is ERC20 {
     constructor(address account, uint256 initialSupply) ERC20("Radicle", "RADICLE") {
         _mint(account, initialSupply.mul(10**uint256(decimals())));
     }
-
-    /// Burn the given amount of tokens from the sender account.
-    function burn(uint256 amount) public {
-        _burn(msg.sender, amount);
-    }
 }
```

Comment:
Will remove this contract later, but for now we need it for the tests and deploy script.